### ISSUE TITLE
Menu.item no longer supports className and onClick?It works in 3.10.x

### ISSUE THREAD

**Issue by AYWK** [ant-design/ant-design#13435]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
I want to simulate a select with menu, which is supported in 3.10.x, but not with 3.11.0。
I see the following code changes：
Its 'props' in 3.10.x is any：
```
class MenuItem extends React.Component<any, any>;
```
but now,it has changed：
```
interface MenuItemProps {
    rootPrefixCls?: string;
    disabled?: boolean;
    level?: number;
    title?: React.ReactNode;/*  */
    children?: React.ReactNode;
}.
```
Why not compatible with 3.10.x?

### What does the proposed API look like?
Menu.item supports className and onClick,Compatible with 3.10 x.

**Comment by sidsethupathi** [comment]
PR https://github.com/ant-design/ant-design/pull/13390 added back in `className`. Should we just open a PR to add back in `onClick` or is there a better solution from the Ant team?

**Comment by zy410419243** [comment]
Fixed in 210aab8d51be70322ef38e0068ac08bae0be0edb
